src/_includes/layouts/home.njk

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```

Comment:
Just an idea to avoid repetition, could this page extend base.njk? You could put all the important landing page stuff in the `{{ block content }}` and keep the same html head and body wrappers. 

you can also use the `{{ block page_title }}` to change the page title.

Reply:
I was running into some issues with extending and Ethan said, that you were working on fixing that so I didn't want to duplicate work. In the meantime, I created a home template that I will switch out after you are done with your changes.

Reply:
Yeah, I was trying to understand why we were using both liquid and njk. But I think its clearer now:
- Templates in the `.njk` will use nunjucks
- Markdown files will use liquid. And the partials called within markdown files should also be written in liquid. 

I was thinking we could just use njk all over, but it'll break a lot of things.

I'd like to know more about what issues you had!

Reply:
Or I can just test this out once its merged

Reply:
Yeah, seems like it wouldn't be a quick change when I looked into it yesterday. It was exactly that, it was complaining about the extend because of it being a markdown template.